test_data/test_configs/bad_aws_key_message_regex.toml

```diff
@@ -7,7 +7,3 @@
     description = "AWS Manager ID"
     regex = '''(A3T[A-Z0-9]|AKIA|AGPA|AIDA|AROA|AIPA|ANPA|ANVA|ASIA)[A-Z0-9]{16}'''
     tags = ["key", "AWS"]
-
-[Global]
```

Comment:
I would think this would cause the tests to break? If not then we should probably remove this test config as it is a valid config now

Reply:
I dont think its necessary this test anymore either but this should have caused the tests to break but it didnt so maybe its something going on with config_tests? I will remove this test

Reply:
All tests were passing but indeed this should have made config tests to break... I have removed the test as there is no point to keep it anymore but maybe we should take a look closer of what happened there

Reply:
Actually I take that back, this should not cause the tests to break since `Global` is being deprecated